src/postgres.rs

```diff
@@ -117,10 +118,35 @@ impl PostgresSession {
         }
         const NUMBER_OF_ARGS: usize = 10;
 
-        let mut args: Vec<&(dyn ToSql + Sync)> = Vec::with_capacity(NUMBER_OF_ARGS * txs.len());
         let txs: Vec<PostgresTransactionInfo> =
             txs.iter().map(PostgresTransactionInfo::from).collect();
+
+        let statement = format!(
+            r#"
+                COPY banking_stage_results.transaction_infos(
+                    signature, errors, is_executed, is_confirmed, first_notification_slot, cu_requested, prioritization_fees, utc_timestamp, accounts_used, processed_slot
+                ) FROM STDIN BINARY
+            "#,
+        );
+        let sink: CopyInSink<bytes::Bytes> = self.copy_in(&statement).await.unwrap();
+        let writer = BinaryCopyInWriter::new(
+            sink,
+            &[
+                Type::TEXT,
+                Type::CHAR,
+                Type::CHAR,
```

Comment:
pls check the types - this is for `is_executed` which is a bool

Reply:
done